pex/pex.py

```diff
@@ -94,6 +94,18 @@ def _extras_paths(cls):
     extras_paths = filter(None, makefile.get('EXTRASPATH', '').split(':'))
     for path in extras_paths:
       yield os.path.join(standard_lib, path)
+    # macOS Sierra support
+    framework = sysconfig.get_config_var("PYTHONFRAMEWORK")
+    if not framework:
+      raise StopIteration()
+    third_party = os.path.join("/Library", framework, sys.version[:3], "site-packages")
+    yield third_party
+    extras_pth = os.path.join(third_party, 'Extras.pth')
```

Comment:
I wonder if it would make sense to classify all .pth-based path additions as "extras" in this context? i.e. to actually look for any files named `*.pth` and parse/emit their fully qualified paths akin to the .pth file handling here: https://github.com/kwlzn/python-sources/blob/master/Python-2.7.12/Lib/site.py#L183-L188 ?

this would more generally handle the case of .pth files injecting new paths that wouldn't be otherwise noticed by pex, with the `Extras.pth`case included.